source/_lovelace/alarm-panel.markdown

```diff
@@ -0,0 +1,72 @@
+---
+layout: page
+title: "Alarm Panel Card"
+sidebar_label: Alarm Panel
+description: "The Alarm Panel allows you to Arm and Disarm your Alarm Control Panel Components"
+date: 2018-10-15 10:28 +00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+---
+
+The Alarm Panel allows you to Arm and Disarm your Alarm Control Panel Components.
+
+<p class='img'>
+<img src='/images/lovelace/lovelace_alarm_panel_card.gif' alt='Screenshot of the alarm panel card'>
+Screenshot of the Alarm Panel card.
+</p>
+
+```yaml
+- type: alarm-panel
+  entity: alarm_control_panel.alarm
+```
+
+{% configuration %}
+type:
+  required: true
+  description: alarm-panel
+  type: string
+entity:
+  required: true
+  description: "Entity ID of `alarm_control_panel` domain"
+  type: string
+title:
+  required: false
+  description: Title of Alarm Panel
+  type: string
+  default: Current State of Alarm Entity
+states:
+  required: false
+  description: Controls which states to have available
+  type: list
+  default: arm_home, arm_away
```

Comment:
Should there be a list somewhere on this page that states valid values for this list?

Reply:
Should that go into the description or somewhere else?

There should indeed be a list somewhere though

Reply:
I guess not in the description but under default

Whats the best way to do this?

Reply:
I have no idea 🤷‍♂️ but I think it should be there somewhere, as a user it would be hard guessing valid values :)

Reply:
Maybe at the bottom?
```
{% linkable_title Valid options for `states` %}

- arm_home
- arm_home
```

Reply:
The usage of `keys` where new to me, but that looks way better 💯